be/src/olap/tablet.cpp

```diff
@@ -3248,6 +3266,82 @@ Status Tablet::commit_phase_update_delete_bitmap(
     return Status::OK();
 }
 
+Status Tablet::full_compaction_update_delete_bitmap(const RowsetSharedPtr& rowset,
```

Comment:
Is it possible to move these methods to FullCompaction class? Too much xx_update_delete_bitmap in Tablet class